README.md

```diff
@@ -42,17 +42,17 @@ For complete pricing details visit our [Chat Pricing Page](https://getstream.io/
 
 ## 🔮 Example Apps
 
-This repo includes 4 example apps. One made with Expo, one Native JavaScript code, and two in TypeScript. One TypeScript app is a simple implementation for reference, the other is a more full featured app example.
+This repo includes 3 example apps. One made with Expo, two in TypeScript. One TypeScript app is a simple implementation for reference, the other is a more full featured app example.
 
 - [Expo example](./examples/ExpoMessaging)
-- [Native example](./examples/NativeMessaging)
 - [Typescript example](./examples/TypeScriptMessaging)
 - [Fully featured messaging application](./examples/SampleApp)
 
 Besides, our team maintains a dedicated repository for fully-fledged sample applications and demos at [GetStream/react-native-samples](https://github.com/GetStream/react-native-samples). Please consider checking following sample applications:
 
 - [Slack Clone](https://github.com/GetStream/react-native-samples/tree/main/projects/SlackClone#slack-clone-using-react-native-and-stream-chat)
 - [iMessage Clone](https://github.com/GetStream/react-native-samples/tree/main/projects/iMessageClone#imessage-clone)
+- [WhatsApp Clone](https://github.com/GetStream/react-native-samples/tree/main/projects/WhatsAppClone#whatsapp-clone-using-react-native-and-stream-chat)
```

Comment:
o_O How is this part of this PR? I would think this would be part of a separate PR? @vanGalilea, did you push something related to this as well?

Reply:
I think `video-attachment` branch needs to be synced with `develop`

Reply:
https://github.com/GetStream/stream-chat-react-native/pull/1383

Reply:
Video upload PR should be merged before merging this PR.